test/sanity/validate-modules/ignore.txt

```diff
@@ -180,7 +180,6 @@ lib/ansible/modules/cloud/cloudstack/cs_vpn_customer_gateway.py E325
 lib/ansible/modules/cloud/cloudstack/cs_zone.py E322
 lib/ansible/modules/cloud/cloudstack/cs_zone.py E325
 lib/ansible/modules/cloud/cloudstack/cs_zone.py E326
-lib/ansible/modules/cloud/digital_ocean/digital_ocean.py E322
```

Comment:
Update the path to match the renamed file instead of removing this line.

Reply:
Thanks @mattclay, once CI failed I realized this should have been changed instead of removed. I was getting ahead of myself and wanting to just removing this module from the project.